lib/mp4/MP4Parser.ts

```diff
@@ -298,6 +301,15 @@ export class MP4Parser extends BasicParser {
   }
 
   private addTag(id: string, value: any) {
+    if (id === "©mvi") {
```

Comment:
@Borewit Some idea to do this better?

In MP4 there are 2 number fields, in ID3 only one (`"x/y"`)

Reply:
Fixed